bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -143,13 +143,20 @@ class PresentationArea extends Component {
   calculateSize() {
     const { presentationHeight, presentationWidth, fitToWidth } = this.state;
     const { currentSlide } = this.props;
-
-    const originalWidth = currentSlide.calculatedData.width;
-    const originalHeight = currentSlide.calculatedData.height;
+    const { calculatedData } = currentSlide || {};
+    const originalWidth = calculatedData.width;
+    const originalHeight = calculatedData.height;
 
     let adjustedWidth;
     let adjustedHeight;
 
+    if (!originalHeight || !originalHeight) {
```

Comment:
Did you mean `!originalWidth || !originalHeight`?

Reply:
Fixed!